src/Components/Artwork/Details/BidTimerLine.tsx

```diff
@@ -45,22 +63,22 @@ export const BidTimerLine: React.FC<BidTimerLineProps> = ({ artwork }) => {
     )
   }
 
-  if (!lotClosesAt || numDays > 5 || biddingEnded) {
+  if (!lotClosesAt || numDays > 5 || hasBiddingEnded) {
     return <EmptyLine />
   }
 
   const renderLotCloseTime = [
     numDays > 0 && `${numDays}d`,
     numHours > 0 && `${numHours}h`,
     numDays === 0 && numHours === 0 && numMinutes > 0 && `${numMinutes}m`,
-    numDays === 0 && numHours === 0 && numMinutes === 0 && `${numSeconds}s`,
+    numDays === 0 && numHours === 0 && numMinutes <= 1 && `${numSeconds}s`,
```

Comment:
@starsirius We should be showing seconds when we're this close (< 2 minutes for the extended bidding period, so it can display '1m 59s, 1m 58s, ... and so on)

Reply:
👍  Nice! I just checked the smaller grid items, like on rails or mobile, and I think the length fits! I'll go back to the design and see if there are any edge case.

![Screenshot 2024-10-06 at 9 49 45 PM](https://github.com/user-attachments/assets/de5b9541-87c0-451e-bfe2-2daa753befee)